content/networks.md

```diff
@@ -0,0 +1,6 @@
+---
+id: Networks
```

Comment:
Why a "s" in the id there? I have the feeling it is one network of mentors, mentees and network managers

Reply:
Thanks for pointing! I have removed it.